.github/workflows/vulnerability_scanning.yml

```diff
@@ -8,23 +8,32 @@ on:
         ## use github hosted runner by default
         default: "ubuntu-latest"
         required: false
+      project:
+        required: true
+        type: string
+      component:
+        required: false
+        type: string
```

Comment:
you don't need `project` `component` as you actually don't care those info in scanning jobs. 

suggest to pass a var `docker_image_tar`